website/docs/r/vpc_dhcp_v4_static_binding_config.html.markdown

```diff
@@ -0,0 +1,121 @@
+---
+subcategory: "VPC"
+layout: "nsxt"
+page_title: "NSXT: nsxt_vpc_dhcp_v4_static_binding"
+description: A resource to configure a DHCP IPv4 Static Binding.
+---
+
+# nsxt_vpc_dhcp_v4_static_binding
+
+This resource provides a method for the management of a DhcpV4StaticBindingConfig.
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+data "nsxt_policy_project" "demoproj" {
+  display_name = "demoproj"
+}
+
+data "nsxt_vpc" "demovpc" {
+  context {
+    project_id = data.nsxt_policy_project.demoproj.id
+  }
+  display_name = "vpc1"
+}
+
+data "nsxt_vpc_subnet" "test" {
+  context {
+    project_id = data.nsxt_policy_project.demoproj.id
+    vpc_id     = data.nsxt_vpc.demovpc.id
+  }
+  display_name = "subnet1"
+}
+
+resource "nsxt_vpc_dhcp_v4_static_binding" "test" {
+  parent_path     = data.nsxt_vpc_subnet.test.path
+  display_name    = "test"
+  description     = "Terraform provisioned DhcpV4StaticBindingConfig"
+  gateway_address = "192.168.240.1"
+  host_name       = "host.example.org"
+  mac_address     = "10:0e:00:11:22:02"
+  lease_time      = 162
+  ip_address      = "192.168.240.41"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `parent_path` - (Required) Policy path of parent VpcSubnet object, typically reference to `path` in `nsxt_vpc_subnet` data source or resource.
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `gateway_address` - (Optional) When not specified, gateway address is auto-assigned from segment configuration.
+* `host_name` - (Optional) Hostname to assign to the host.
+* `mac_address` - (Optional) MAC address of the host.
+* `lease_time` - (Optional) DHCP lease time in seconds.
+* `ip_address` - (Optional) IP assigned to host. The IP address must belong to the subnet, if any, configured on Segment.
+* `options` - (Optional) None
+  * `option121` - (Optional) None
+    * `static_route` - (Optional) Classless static route of DHCP option 121.
```

Comment:
`Required` here and below in `code`

Reply:
Yep. Also network, gateway should be required.
BTW do you think that we should require these also in vpc_subnet? Everything is optional there.

Reply:
Yes, I think so